services/website/content/en/stake-pools/2c3390339aba47100f9ab1a230d6d726e10e9d527be49dc5bab479da.md

```diff
@@ -5,4 +5,59 @@ memberSince: 2021-07-02
 identities:
   - id: saad
     role: spo
+  - id: wasif
+    role: spo
+twitter: rubypools
+github: ruby-pool
+nodes:
+  - role: producer
+    name: Ruby Producer
+    deviceId: cloud-server-8gb
+    storage: SSD 80GB
+    isBackup: false
+  - role: relay
+    name: Ruby Relay
+    deviceId: cloud-server-8gb
+    storage: SSD 80GB
+    isBackup: false
+  - role: relay
+    name: Ruby Relay
+    deviceId: cloud-server-8gb
+    storage: SSD 80GB
+    isBackup: false
```

Comment:
`deviceId` must be removed for each node. Once you're over on actual Raspberry Pi's you'll be able to add `deviceId: raspberry-pi-4-8gb`.